client/src/App.jsx

```diff
@@ -33,26 +32,24 @@ class App extends React.Component {
 
   //  on previous click changes state of currpage/prev page and grabs new pictures
   onPrevClick() {
-    if (this.state.prevPage !== 0) {
+    if (this.state.currPage !== 1) {
 
       const cPage = this.state.currPage - 1;
-      const pPage = this.state.prevPage - 1;
+      const pPage = this.state.currPage - 2;
       this.setState({
         currPage: cPage,
-        prevPage: pPage,
         showAdFeedBack: false,
       });
-      this.getPagePictures(cPage, pPage, false);
+      this.getPagePictures(cPage, pPage);
       this.renderAdShow();
     } else {
       const cPage = this.state.maxPages;
       const pPage = this.state.maxPages - 1;
       this.setState({
         currPage: cPage,
-        prevPage: pPage,
         showAdFeedBack: false,
       });
-      this.getPagePictures(cPage, pPage, false);
+      this.getPagePictures(cPage, pPage);
```

Comment:
you already know what i'm going to say and it is lol 1. definitely could refactor this stuff. on prev and on next are the same function, they're just updating the state in opposite directions. You could pass that part down in a prop and refactor them into one super quick! DRY it down